search-ui/src/bcrs-common-components/resources/breadcrumb-resources.ts

```diff
@@ -0,0 +1,17 @@
+import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
+
+
+export const SearchHomeBreadCrumb: BreadcrumbIF = {
+    text: 'BC Search and Online Services',
+    href: sessionStorage.getItem('REGISTRY_URL')
+}
+
+export const SearchDashboardBreadcrumb: BreadcrumbIF = {
+    text: 'BC Search Dashboard',
+    href: '/dashboard'
```

Comment:
instead of href this should be `to: { name: RouteNames.DASHBOARD }` right?

Reply:
Updated. Thanks!